test/Microsoft.DotNet.Tools.Tests.Utilities/Commands/CacheCommand.cs

```diff
@@ -83,7 +85,7 @@ private string BuildArgs()
                 FrameworkVersionOption);
         }
 
-        private string ProfileProjectOption => string.IsNullOrEmpty(_profileProject) ? "" : $"--entries {_profileProject}";
+        private string ProfileProjectOption => _profileProject.Count == 0  ? "" :  string.Join(" ", _profileProject) ;
```

Comment:
Also, each profile project needs to have `--entries` on it.  That's how multi-value options work.

---
In reply to: [104684104](https://github.com/dotnet/cli/pull/5942#discussion_r104684104) [](ancestors = 104684104)